nowcasting_dataset/manager.py

```diff
@@ -180,23 +180,38 @@ def create_files_specifying_spatial_and_temporal_locations_of_each_example_if_ne
             ],
         )
         for split_name, datetimes_for_split in split_t0_datetimes._asdict().items():
-            n_batches = self._get_n_batches_for_split_name(split_name)
-            n_examples = n_batches * self.config.process.batch_size
+            path_for_csv = self.config.output_data.filepath / split_name
+            n_batches_requested = self._get_n_batches_requested_for_split_name(split_name)
+            if (n_batches_requested == 0 and len(datetimes_for_split) != 0) or (
+                len(datetimes_for_split) == 0 and n_batches_requested != 0
+            ):
+                msg = (
```

Comment:
good clear message

I ahve see {variable=} and thanks prints out variable={variable} as a short hand

Reply:
ooooh!  `f"{variable=}"` is a really nice trick!  Thanks loads for telling me about that!  I'm gonna use that all over the place now :slightly_smiling_face: